### DISCUSSION TITLE
How to get main image form a link?

### DISCUSSION THREAD

**Discussion by SatyamDevv** [burhanrashid52/PhotoEditor#461] [Q&A]
I am try to get main image for link like this
> Glide.with(MainActivity.this).load(imageLink).into(photoEditorView);

It give error like:-

Bound mismatch: The generic method into(Y) of type RequestBuilder<TranscodeType> is not applicable for the arguments (PhotoEditorView). The inferred type PhotoEditorView is not a valid substitute for the bounded parameter <Y extends Target<Drawable>>

**Comment by burhanrashid52** [comment]
`PhotoEditorView` is a `ViewGroup`. You can't use that as target in Glide. Instead you should `ImageView` return form `PhotoEditorView` like this.
```kotlin
Glide.with(MainActivity.this).load(imageLink).into(photoEditorView.getSource());
```

**Reply by SatyamDevv** [reply]
Thx very much sir
